checker-framework-eclipse-update-site/manual/index.html

```diff
@@ -0,0 +1,338 @@
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
I would not commit this html file and the images. Instead, we should work on an up-to-date manual in a more suitable format, e.g. markdown.

Reply:
Removed the manual folder.